code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -267,7 +267,7 @@ GLOBAL_LIST_INIT(titaniumglass_recipes, list(
 
 /obj/item/stack/sheet/titaniumglass/examine_more(mob/user)
 	. = ..()
-	. += "Titanium-silicate mixes are an old but highly effective technology that produce a relatively lightweight, very strong glass that can withstand a good amount of punishment."
+	. += "Titanium-silica mixes are an old but highly effective technology that produce a relatively lightweight, very strong glass that can withstand a good amount of punishment."
```

Comment:
This isn't really part of the fix, `silicate` is a legit naming convention https://www.britannica.com/science/silicate-mineral